_data/providers.yml

```diff
@@ -261,6 +261,15 @@ providers:
     sms: Yes
     software: Yes
     url: https://www.vasco.com/
+	
```

Comment:
There's tab character on blank line. In general our yaml doesn't allow tabs, only spaces.